pages/index.html

```diff
@@ -5,66 +5,70 @@
-                                <span class="h5">Latest version: {{ latest_stable.version }}</span>
-                                </a>
-                            </li>
-                            <li id="banner-wiki"> <a href="https://wiki.openttd.org/">
-                                <span class="h3">Learn&nbsp;how&nbsp;to&nbsp;play</span>
-                                <span class="h5">Read our wiki for help</span>
-                                </a>
-                            </li>
-                            <li id="banner-forum"> <a href="https://forum.openttd.org/">
-                                <span class="h3">Visit&nbsp;our&nbsp;community</span>
-                                <span class="h5">Join us on the forums</span>
-                                </a>
-                            </li>
-                        </ul>
-                    </div>
-                    <div id="openttd-banner">
-                        <div id="openttd-text">OpenTTD is an open source simulation game based upon Transport Tycoon Deluxe</div>
-                    </div>
-                </div>
-            </div>
-            <div id="index-right">
-                <div id="screenshot">
-                    {% assign random_screenshot = site.screenshots | sample %}
-                    <div id="screenshot-image">
-                        <a href="{{ site.baseurl }}/screenshots.html">
-                            {% assign stripped_content = random_screenshot.content | strip_html | strip_newlines %}
-                            <img src="{{ site.baseurl }}{{ random_screenshot.id }}_thumb.png"
-                                alt="{{ stripped_content }}"
-                                title="{{ stripped_content | truncatewords: 5 }}"
-                                width="232"
-                                height="178"
-                                />
-                        </a>
-                    </div>
-                    <div id="screenshot-content">
-                        {{ random_screenshot.content }}
-                    </div>
-                </div>
+<div id="content">
+    <div id="openttd-banner-left-{{ "now" | date: "%u" }}" class="index-banner">
+        <img id="openttd-banner" src="{{ site.staticurl }}/img/layout/openttd-128.gif" />
+        <div id="openttd-text">OpenTTD is an open source simulation game based upon Transport Tycoon Deluxe</div>
+        <div id="banner-links">
+            <ul>
+                {% assign latest_stable = site.downloads | where_exp: "download", "download.id contains '/downloads/openttd-releases/latest'" | last %}
+                <li id="banner-download">
+                    <a href="{{ site.baseurl }}{{ latest_stable.url }}">
+                        <img src="{{ site.staticurl }}/img/layout/index-download.gif" />
+                        <div class="banner-text">
+                            <h3>Download OpenTTD</h3>
```

Comment:
It's IMHO not a good idea to put the text fragments of a link into headers (`<hx>`) especially when a `h3` is directly followed by a `h5`.

Reply:
It's always seemed silly to me to fake those elements with CSS classes though. The elements are there, why not use them?

Reply:
Headings (`h1` to `h6`) should reflect a documents semantic structure. The heading elements are not intended to make it easier to format text fragments.

I'll make a proposal in a further PR *especially for the list with these three links*, trying to keep the simpler HTML-structure.

Reply:
Very good, I'll put them back for now